lib/src/chunk.dart

```diff
@@ -5,43 +5,7 @@ import 'fast_hash.dart';
 import 'marking_scheme.dart';
 import 'nesting_level.dart';
 import 'rule/rule.dart';
-
```

Comment:
This was just moved to a separate file so that the new code doesn't import `chunk.dart`, which is part of the old IR.